cdap-ui/app/cdap/services/StatusMapper.js

```diff
@@ -47,21 +47,27 @@ function lookupDisplayStatus (systemStatus) {
 }
 
 function getStatusIndicatorClass (displayStatus) {
-  if (displayStatus === 'Running' || displayStatus === 'Starting') {
+  if (displayStatus === 'Running' || displayStatus === 'Starting' || displayStatus === 'Pending') {
     return 'status-blue';
-  } else if (displayStatus === 'Succeeded' || displayStatus === 'Starting' || displayStatus === 'Scheduling' || displayStatus === 'Stopping') {
+  } else if (displayStatus === 'Succeeded' || displayStatus === 'Scheduling' || displayStatus === 'Stopping') {
     return 'status-light-green';
   } else if (displayStatus === 'Failed') {
     return 'status-light-red';
-  } else if (displayStatus === 'Draft') {
-    return 'status-outline-grey status-thin';
   } else {
     return 'status-light-grey';
   }
 }
 
+function getStatusIndicatorIcon (displayStatus) {
+  if (displayStatus === 'Pending' || displayStatus === 'Draft') {
```

Comment:
Sorry I missed this earlier but can we use the `StatusMapper` for the status of the pipeline? Will reduce the amount of things we hardcode.